pandas/core/generic.py

```diff
@@ -1591,62 +1591,87 @@ def _repr_latex_(self):
     # I/O Methods
 
     _shared_docs['to_excel'] = """
-    Write %(klass)s to an excel sheet
-    %(versionadded_to_excel)s
+    Write %(klass)s to an excel sheet.
+
+    To write a %(klass)s to an excel .xlsx file it is necessary to first create
+    an ExcelWriter object with a target file name, and specify a sheet in the
+    file to write to. Multiple sheets may be written to by
+    specifying unique sheet_name. With all data written to the file it is
+    necessary to save the changes. Note that creating an ExcelWriter object
+    with a file name that already exists will result in the contents of the
+    existing file being erased.
 
     Parameters
     ----------
     excel_writer : string or ExcelWriter object
-        File path or existing ExcelWriter
+        File path or existing ExcelWriter.
```

Comment:
Can we link to the class? I believe
```
:class:`pandas.ExcelWriter`
``` 
should work.

Reply:
Also, in the extended description we say that an `ExcelWriter` object is necessary to be able to use `.to_excel`, but in the parameter description we say that we also accept a "file path". So the `ExcelWriter` is not really needed? What happens if we specify a file path? I guess that a new `ExcelWriter` gets automatically created with default options.

Reply:
Yes this is allowed `df1.to_excel("example.xlsx", sheet_name='Sheet3')`

Reply:
Yes, the ExcelWriter is not required